cmd/xdpderper/xdpderper.go

```diff
@@ -18,6 +18,7 @@ import (
 	"tailscale.com/derp/xdp"
 	"tailscale.com/net/netutil"
 	"tailscale.com/tsweb"
+	_ "tailscale.com/tsweb/promvarz"
```

Comment:
Maybe:

```suggestion

	// Include support for native Prometheus histograms.
	_ "tailscale.com/tsweb/promvarz"
```

(Rationale: Except when it's obvious like "embed", I like to comment blank imports since the compiler can't explain why they're present)

Reply:
Done.